atomics/T1078.004/T1078.004.md

```diff
@@ -112,7 +112,7 @@ Automation runbook to execute malicious code and maintain persistence in their t
 ```powershell
 $secure_pwd = "#{password}" | ConvertTo-SecureString -AsPlainText -Force
 $creds = New-Object System.Management.Automation.PSCredential -ArgumentList "#{username}", $secure_pwd
-Connect-AzAccount -Credential $creds
+Connect-AzAccount -Credential $creds  # NOTE: Will not work if the test Azure account has 2FA enabled. Only works if account does not have 2FA. Workaround is to run `Connect-AzAccount` and login via the browser, with MFA.
```

Comment:
Hello @kienmarkdo ! Can you make this change in the associated YAML file. These .md files are auto generated from the yaml. 

Also, can we make this note in the description of the yaml instead of the command section? That is much cleaner way to get this information embedded!

Reply:
Done and ready for review.